packages/core/src/html/includePanelProcessor.js

```diff
@@ -228,6 +230,12 @@ function processInclude(node, context, pageSources, variableProcessor, renderMd,
     }
   }
 
+  if (shouldOmitFrontmatter) {
+    const $ = cheerio.load(actualContent);
```

Comment:
let's modify the [`Context` class](https://github.com/MarkBind/markbind/blob/master/packages/core/src/html/Context.js) which contains the parent -> child -> child -> ... "context" instead, and the `frontmatter` processing function

`cheerio.load` is a rather expensive operation (parses the entire page again)

Reply:
Ah I see. Okay, will try to do that.

Reply:
I have updated the approach. Just checking, is it okay if I put the `omitFrontmatter` flag in the `variables` property of the context? I'm afraid on whether it will pollute site/page variables inside.

Reply:
how about adding a new property to the `Context` class? should avoid the concern you mentioned here

Reply:
Alright. I will add another property `processingOptions` (or something along those lines) that will have the `omitFrontmatter` flag (and other processing options if something to be added in the future)